tests/loader_wsi_tests.cpp

```diff
@@ -310,6 +350,46 @@ TEST_F(WsiTests, GetPhysicalDeviceXcbPresentICDSupport) {
 
     ASSERT_EQ(VK_TRUE, env->vulkan_functions.vkGetPhysicalDeviceXcbPresentationSupportKHR(physical_device, 0, nullptr, 0));
 }
+
+TEST_F(WsiTests, XcbGetPhysicalDeviceSurfaceSupportKHR) {
+    FrameworkEnvironment env{};
```

Comment:
TEST_F(WsiTests, provides a framework environment, so this is going to cause confusion as two test frameworks are vying for control.

Reply:
So my tests failed if I mixed `TEST_F` with `TEST`, so to resolve this, I converted all the WSI tests to `TEST`.